docs/courses/csintro3/orientation/math.md

```diff
@@ -1,101 +1,171 @@
+
+While parenthesis are not an operator, they serve as a way to group operations together. Operations done within parenthesis will be completed first before moving onto those outside.
+
+If we look at the example
 
 ```typescript
-let num: number = 0;
-num += 1;
-num += 1;
-num += -1;
-num += 1;
+let num: number = 24 / 3 * 4;
+console.log("" + num)
 ```
 
+This will print out the number 32 because the division occurs before the multiplication. If we want to change this order and multiply before we divide, we can use parenthesis like
+
 ```typescript
-let num: number = 0;
-num++;
-num++;
-num--;
-num++;
+let num: number = 24 / (3 * 4)
+console.log("" + num)
 ```
 
-Explain why -> very commonly used, easier to write as a short hand, etc
+This will output 2 instead
 
-## Other assignment operators (quick over view as less commonly used at first)
 
-### minus equals
+## Student Task 1:
 
-```typescript
-let num: number = 15;
-num = num - 5;
-```
+1. Pick 4 random numbers and write them horizontally on piece of paper
+2. In the 3 spaces between those numbers, place a random operator to put there
+3. Evaluate the expression on paper
+4. Type out the expression in JavaScript like the above examples and compute the result
+
+## Assignment Operators
+
+There are many instances in which you may want to set a variable to an expression that references itself, such as
 
 ```typescript
-let num: number = 15;
-num -= 5;
+num = num + 5;
 ```
+When programming, there is actually a way to condense this expression that makes it easier to write.
 
-### times equals
+The above example can also be written as
 
 ```typescript
-let num: number = 15;
-num = num * 5;
+num += 5;
 ```
 
-```typescript
-let num: number = 15;
-num *= 5;
+**Note:** This actually how the ``||variables:change by||`` block works so the above is equivalent to 
+
+```block
+let num = 0
+num += 5
 ```
 
-### divided by equals
+This process of condensing the expression extends to all operators, so
 
-```typescript
-let num: number = 15;
-num = num / 5;
+`num = num - 5;` is equivalent to `num -= 5;`
+
+
+`num = num * 5;` is equivalent to `num *= 5;`
+
+
+`num = num / 5;` is equivalent to `num /= 5;`
+
+
+### Special case: add or subtract 1 (increment / decrement operators)
+
+Another common thing done in JavaScript is to increase or decrease a value by 1. 
+
+We've seen that you can do this by saying either
+
+`num = num + 1;` or `num += 1;`
+
+We can also say `num++;`
+
+All three of these are valid lines that would increment the value by 1.
+
+It also extends to decreasing a value by 1 in that `num--` will decrement the value of `num` by 1.
+
+These two increment/decrement operators are very commonly used as they are quicker and easier to read and write
+
+
+## Math Functions / calling functions intro
+
+When programming in blocks, there were a few Math/number related functions that we called. Things such as
```

Comment:
I'd drop the '/number' here, all of them are from the math category anyway. Also, generally style in course has been separated `/`'d things; e.g. 'Math / number'